src/components/layout/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import AccountPopup from './account-popup';
+import styles from './styles.module.css';
```

Comment:
```suggestion
import styles from './layout.module.css';
```
also, please change the respective file name to layout.module.css